### PR TITLE
Dp24 gc content

### PR DESCRIPTION
**Pull request by DLBPointon**
Adds GC_CONTENT and GENERATE_GENOME as well as respective modules and scripts

### CODE REVIEW

File: workflows/ascc.nf
Comment (weaglesBio) [inline workflows/ascc.nf:1]: Add versions GC_CONTENT versions mix

**Review by weaglesBio — COMMENTED** [review]
Minor changes to version tracking, otherwise good.

File: subworkflows/local/generate_genome.nf
[the remark this answers is not in the document]
Comment (DLBPointon) [reply subworkflows/local/generate_genome.nf:1]: Oooo I copy-pasted this from TreeVal, I'll add that to the bugs.

File: modules/local/gc_content.nf
[the remark this answers is not in the document]
Comment (DLBPointon) [reply modules/local/gc_content.nf:1]: If it's a tool in the container then I think it should get recorded, especially something like pandas where it is a very large data science package and sometimes specific versions are needed.

[the remark this answers is not in the document]
Comment (weaglesBio) [reply modules/local/gc_content.nf:1]: Okay, in that case it is missing from the gc_content.nf

**Review by weaglesBio — COMMENTED** [review]
Version mixes are okay, pandas and python version should be included in the versions.yml in gc_content.nf

**Comment by DLBPointon** [comment]
> Version mixes are okay, pandas and python version should be included in the versions.yml in gc_content.nf


I'll get on that Monday, I'll copy it across to other modules too

**Comment by DLBPointon** [comment]
@weaglesBio 
Morning Will,
Only get_lineages_kraken actually uses pandas, so I've corrected the container in use as well as the version data output by the modules.

File: bin/reformat_diamond_outfmt6.py
Comment (yumisims) [inline bin/reformat_diamond_outfmt6.py:1]: consider to change : print(f"{s[0]}\t{s[4]}\t{s[5]}\t{s[6]}\t{s[7]}\t{s[8]}\t{s[9]}\t{s[10]}\t{s[11]}\t{s[12]}\t{s[13]}\t{s[2]}") 
to:
print('\t'.join( s[0:1]  + s[4:] + s[2:3] ))
?

**Review by yumisims — APPROVED** [review]

File: bin/reformat_diamond_outfmt6.py
Comment (DLBPointon) [reply bin/reformat_diamond_outfmt6.py:1]: I've added this in, thank you.

**Pull request merged**